code/modules/modular_computers/computers/machinery/console_presets.dm

```diff
@@ -39,6 +39,7 @@
 	hard_drive.store_file(new/datum/computer_file/program/alarm_monitor())
 	hard_drive.store_file(new/datum/computer_file/program/supermatter_monitor())
 
+
```

Comment:
```suggestion
```

Reply:
Huh? What does that change, exactly? I'm still new to GitHub.

Reply:
lmao it doesn't actually affect how your code works. it's just convention to have consistent spacing between chunks of code for readability. This changes it from 2 blank lines to 1

Reply:
Oh, okay, it's just that one of them had two, so I thought I'd make them all have two. My bad, will commit.